decidim-core/config/locales/en.yml

```diff
@@ -1484,9 +1484,9 @@ en:
       floating_help:
         help: Help
       follow_button:
-        sign_in_before_follow: Please sign in before performing this action
+        sign_in_before_follow: Please log in before performing this action
```

Comment:
```suggestion
        log_in_before_follow: Please log in before performing this action
```

Reply:
@alecslupu I'm reverting this change back as its breaking tests. The most important aspect I believe is that the value reflects the change in the change, not the key.

Reply:
@greenwoodt yes, i breaking tests, but most of the tests are broken with
```
ActionView::Template::Error:
       Translation missing: en.decidim.shared.follow_button.sign_in_before_follow
```
if you find the place where string is added, then the issue could be easily solved.